examples/cpp_sycl/makefile_lnx

```diff
@@ -24,7 +24,7 @@ help:
 	@echo
 	@echo "name              - example name. Please see daal.lst file."
 	@echo
-	@echo "compiler_name     - can be clang, gnu or intel. Default value is clang."
+	@echo "compiler_name     - can be clang. Default value is clang."
```

Comment:
the wording is a bit off
I'd change it to "the only supported compiler is clang (default value)"

Reply:
fixed